tests/test_tpt_reports.py

```diff
@@ -135,3 +135,29 @@ def test_domain_validation():
         except SystemExit as sys_exit:
             return_code = sys_exit.code
             assert return_code == 2, "main() should return with error return code 2"
+
+
+def test_parse_json():
+    """Validate parse json functionality."""
```

Comment:
Make it clearer that this test applies to the `parse_json()` function.
```suggestion
    """Validate parse_json() functionality."""
```